lib/middleware/index.js

```diff
@@ -25,19 +25,12 @@ Login.prototype.middleware = function(store, options) {
   if (typeof this.options.resetPasswordTimeLimit === 'string')
     this.options.resetPasswordTimeLimit = ms(this.options.resetPasswordTimeLimit);
 
-  this.store = store;
+  this.store = backend;
```

Comment:
There are places left where store needs to be renamed. Only after we're sure all places have been updated, it can be removed. When I did the PR I unfortunately didn't have the time (nor need) to fix all those issues (there are also a bunch of tests failing) reliably, so I thought this be good start anyway.